tlcc_enclave/README.md

```diff
@@ -11,9 +11,12 @@ the blockchain state.
 
 ## Start with generating proto parser
 
-We use *nanopb*, a lightweight implementation of Protocol Buffers, inside the
-ledger enclave to parse blocks. Install nanopb by following the instruction on
-http://github.com/nanopb/nanopb
+We use *nanopb*, a lightweight implementation of Protocol Buffers, inside the ledger enclave to parse blocks of
+transactions. Install nanopb by following the instruction below. For more detailed information consult the official
+nanopb documentation http://github.com/nanopb/nanopb.
+
+[comment]: # (note by christina: I would rather write: "Install nanopb by the following instructions. Because one does not have to go through the whole README of
```

Comment:
the comment seems obsolete now?  (But i agree with rewording: official README is actually more confusing but it does _not_ make obvious the actual steps you have to do ...)

Reply:
ok, removed the comment